### ISSUE TITLE
[Nova Função] Cadastrar categoria google merchant para toda a Loja de uma vez

### ISSUE THREAD

**Issue by rlasmar** [opencartbrasil/opencartbrasil#20]
Área a seguir:  Alimentadores - Google Merchant Center

Para cada departamento, é necessário configurar a categoria do google. 

Para casos em que a loja só vende em uma categoria do google, o ideal é configurar todos os departamentos de uma vez só em uma categoria.

Tenho 120 departamentos na minha loja, e para cada departamento estou tendo que configurar a mesma categoria do google.

Gostaria de solicitar uma opção para:
Escolher a categoria e escolher todos os departamentos de uma vez. Toda a loja na mesma categoria.

![Capturar](https://user-images.githubusercontent.com/37814975/62268149-dc42b300-b3fc-11e9-8569-a631bb467fdc.JPG)

**Comment by rlasmar** [comment]
No Opencart 2 basta colocar URL do google base que gerava o XML. No opencart ele só gera o XML com os produtos se cadastrar cada categoria, certo?

**Comment by v5mvidal** [comment]
Neste caso seria interessante de fato ter uma opção no módulo **Google Merchant Center** que permita selecionar apenas uma categoria e informar que ela é a categoria padrão de todos os departamentos, assim creio que resolveria, o que você acha?

Seu XML para Google Merchant Cente está funcionando normalmente?

**Comment by rlasmar** [comment]
Sim, o XML está funcionando, porém tive que adicionar todas os 165 departamentos manualmente. E a cada vez que adiciono novo departamento em meu site, tenho que lembrar de adicionar manualmente no Google Merchant. Já esqueci várias vezes e os produtos não estavam indo para o Merchant.

E minha loja só tem uma categoria do google, então é exatamente o que preciso, selecionar uma categoria do google merchant e que todas os departamentos sejam associados a esta categoria, ao invés de adicionar um por um.
